cloud/azure/deploytf/.nitric/modules/website/outputs.tf

```diff
@@ -0,0 +1,7 @@
+# output "uploaded_files" {
+#   value = {
+#     for file, blob in azurerm_storage_blob.website_files :
+#     file => blob.url
+#     if blob.content_md5 != filemd5("${var.local_directory}/${file}") # Filter changed files
+#   }
+# }
```

Comment:
can probably remove this or leave a comment on the conditions that are required to implement this

Reply:
Done

Reply:
implemented